frontend/lib/src/components/elements/Toast/Toast.test.tsx

```diff
@@ -97,7 +97,7 @@ describe("Toast Component", () => {
     const expandButton = screen.getByRole("button", { name: "view more" })
     expect(toast).toBeInTheDocument()
     expect(toast).toHaveTextContent(
```

Comment:
Nit: Would prefer a more specific selector to do the `toHaveTextContent` check on:
```javascript
// alert role covers both the StreamlitMarkdown & the view more/close buttons
const toast = screen.getByRole"alert")
// Could use the within selector to get more specific
const toastText = within(toast).getByTestId("stMarkdownContainer")
```